client/me/purchases/purchase-item/index.jsx

```diff
@@ -174,6 +174,7 @@ class PurchaseItem extends Component {
 		const { isPlaceholder, isDisconnectedSite, purchase } = this.props;
 		const classes = classNames(
 			'purchase-item',
+			{ 'is-disconnected-site': isDisconnectedSite }, // Required for e2e tests, see https://github.com/Automattic/wp-e2e-tests/pull/1349
```

Comment:
The comment helps, but I'd prefer to keep classes for styling.

How about using a data attribute for targeting these items? There's precedent around Calypso for using `data-e2e-…`.